map:
  bounds_min: [-50, -50, 0   ]
  bounds_max: [ 50,  50, 15.0]
  get_from_param: false
  obstacles: [
              {
                "type": 0,
                "size": [25.0, 0.0, 0.0],
                "center": [0.0, 0.0, 25.0],
                "rpy": [0.0, 0.0, 0.0]
              },
              {
                "type": 1,
                "size": [50.0, 50.0, 150.0],
                "center": [100.0, 0.0, 75.0],
                "rpy": [0.0, 0.0, 0.0]
              }
              ]

  random_map:
    obs_count: 45
    radius_min: 2.0
    radius_max: 5.0

planner:
  # 0 - RRT 
  # 1 - RRTStar
  # 2 - RRTConnect
  # 3 - InformedRRTStar
  # 4 - BITStar
  # 5 - PRMStar
  # 6 - RRTXStatic
  type: 4
  
  # 0 - R^3
  # 1 - R^3 x SO(2)
  # 2 - R^3 x SO(3)
  state_space: 0
  state_interp_res: 0.30
  plan_time: 0.40
  use_full_planning_time: false
  wait_for_exact_soln: true
  smoothen: false
  auto_range: false
  range: 5.0
  virt_ceil_hgt: 10.0
  ground_clearance: 0.0
  optim:
    en_clear_obj: false
    k_clear: 1.0
    k_length: 1.0
  
  # mav_shape:  {
  #               "type": 3,
  #               "size": [5.0, 1.0, 1.0],
  #               "center": [0.0, 0.0, 0.0],
  #               "rpy": [0.0, 0.0, 0.0]
  #             }

  mav_shape:  {
                "type": 0,
                "size": [0.50, 1.0, 1.0],
                "center": [0.0, 0.0, 0.0],
                "rpy": [0.0, 0.0, 0.0]
              }


  goal_z: 2.0
  start_z: 2.0


octomap_interface:
  map_res: 0.50